docs/application/native/guides/device/battery-monitor.md

```diff
@@ -79,90 +77,105 @@ To get the battery usage information related to the application ID:
     }
     ```
 
-- Get total battery usage of an application ID over a specific duration of time.
-  To get a particular time duration type, use the `battery_monitor_duration_type_e` enum:
+- Get the total battery usage of an application ID over a specific interval of time. The timestamps **from** and **to** must be set as per the Unix Epoch time format:
 
     ```
-    duration_val = BATTERY_MONITOR_DURATION_TYPE_1DAY;
-    error_code = battery_monitor_get_total_usage_by_app_id(app_id, duration_val, &battery_usage);
+    /* To get the usage for past 24 hours from the current epoch time */
+    int period_t = 24*60*60; //24 hours in seconds
+    to = now; from = to - period_t;
+
+    error_code = battery_monitor_get_power_usage_by_app(app_id, from, to, &battery_usage);
     if (error_code == BATTERY_MONITOR_ERROR_NONE)
-        printf("The total battery usage for appid [%s] over last 24 hrs is [%d]", app_id, battery_usage);
+        printf("The Total Battery Usage for appid [%s] over last 24 hrs is [%lf]", app_id, battery_usage);
```

Comment:
@nawab-samsung Question. According to your other comments, isn't "by" better than "for"?

The Total Battery Usage _by_ appid [%s] ...

Reply:
@mijin85cho  I think appid is a value(a reference to an app/ index of an app),  but not the app itself, so it( the value itself) may not consume battery that why for is appropriate. My understanding can be wrong, so kindly give your valuable suggestion, will go as you say.

I do agree that "by" gives better clarity but it is situation based, 
Also you had commented with your suggestion https://github.com/Samsung/tizen-docs/pull/1076#discussion_r409361830. 
So kept it intact. 
@d1-agarwal What are your thoughts on this?

Reply:
@d1-agarwal  kindly keep the current update as is.

Refer to the discussion: 
https://github.com/Samsung/tizen-docs/pull/1076/files#r414306294
https://github.com/Samsung/tizen-docs/pull/1076/files#r414417039